dgisim/src/agents.py

```diff
@@ -115,6 +120,24 @@ def choose_action(self, history: List[GameState], pid: GameState.Pid) -> PlayerA
                             )
                         )
                     )
+                
+            # starsigns
+            if cards.contains(Starsigns):
+                card = Starsigns
+                tmp_dices = ActualDices({Element.ANY: 2})
+                print(card().name())
+                return CardAction(
+                    card,
+                    CharacterTargetInstruction(
+                        tmp_dices,
+                        StaticTarget(
+                            pid,
+                            Zone.CHARACTER,
+                            active_character.get_id(),
+                        )
+                    )
+                )
```

Comment:
just tested in Genshin with npc, player cannot play Starsigns when the active character has their energy full. pls add the corresponding condition here. (just nitpick)

Reply:
fixed in latest push